package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve --port 4201",
+    "start": "ng serve --port 4201 -c development",
```

Comment:
I think we can leave it as `start` since this is the script for starting the development server. We would never use it in production environment since there we are serving a static production build via ngnix or however we do that.